hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/KeyValueHandler.java

```diff
@@ -546,6 +547,43 @@ ContainerCommandResponseProto handleGetCommittedBlockLength(
     return getBlockLengthResponse(request, blockLength);
   }
 
+  /**
+   * Handle List Block operation. Calls BlockManager to process the request.
+   */
+  ContainerCommandResponseProto handleListBlock(
+      ContainerCommandRequestProto request, KeyValueContainer kvContainer) {
```

Comment:
I think we should require container token for `ListBlock` operation:

https://github.com/apache/ozone/blob/e6179c1ec6d77a46687c513ab63b81d7c62e9796/hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/HddsUtils.java#L428-L440